src/actions/article.actions.js

```diff
@@ -53,6 +61,19 @@ export const postArticle = data => {
   };
 };
 
+export const rateArticle = data => {
+  return async dispatch => {
+    try {
+      const { data: article } = await http.post('/ratings', data);
+      toast.success(article.message);
+      return dispatch(postRatingSuccess());
```

Comment:
Great work here. However, I don't think we need to return anything in this action

Reply:
Thanks for the feedback, I have removed the return statement.